app/src/main/java/swati4star/createpdf/Home.java

```diff
@@ -272,36 +304,38 @@ public void onClick(View v) {
         }
     }
 
-
     // Create Pdf of selected images
     void createPdf() {
         if (imagesUri.size() == 0) {
-            Toast.makeText(activity, R.string.toast_no_images, Toast.LENGTH_LONG).show();
-        } else {
-            new MaterialDialog.Builder(activity)
-                    .title(R.string.creating_pdf)
-                    .content(R.string.enter_file_name)
-                    .input(getString(R.string.example), null, new MaterialDialog.InputCallback() {
-                        @Override
-                        public void onInput(MaterialDialog dialog, CharSequence input) {
-                            if (input == null || input.toString().trim().equals("")) {
-                                Toast.makeText(activity, R.string.toast_name_not_blank, Toast.LENGTH_LONG).show();
-                            } else {
-                                filename = input.toString();
-
-                                new CreatingPdf().execute();
-
-                                if (mMorphCounter1 == 0) {
-                                    mMorphCounter1++;
-                                }
+            if (tempUris.size() == 0) {
```

Comment:
why nested if on same condition.

Reply:
It's not the same condition. I will first check whether the imageUri List is empty and then, in case the tempUri List isn't empty, clone the tempUri List to the imageUri List, and show the "no images selected" toast in case both lists are empty

Reply:
Sorry. My bad. I thought these were same variables.